resources/js/components/global/Dashboard.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div class="test">
+    <DashboardNotification></DashboardNotification>
+    <div class="content mt-5">
+      <div class="row justify-content-center" v-for="(item, index) in headings" :key="index">
```

Comment:
The div is overflowing horizontally because `.row` attaches a left and right margin. What you could possibly do is set the div to `width: 100%` since it will include margin in calculation with `box-sizing: content box;` from our overrides; or you can set `margin: 0`.